README.md

```diff
@@ -207,6 +207,24 @@ pixi run test
 The project will be built in the `build` directory.
 You can run `pixi shell` and build the project with `cmake` and `ninja` manually.
 
+## Release with Pixi
```

Comment:
I would suggest moving specialized topics to a dedicated readme section to clear the current one.
We can put many things in release.md for instance.

Reply:
Ok, did using the development sub directory is fine to you ?

We can have the following file inside
- development/release.md
- development/scripts/pixi/activate.*
- development/contributing.md

Reply:
Perfect